yolov5/train.py

```diff
@@ -142,44 +112,15 @@ def train(hyp,  # path/to/hyp.yaml or hyp dictionary
     nc = 1 if single_cls else int(data_dict['nc'])  # number of classes
     names = ['item'] if single_cls and len(data_dict['names']) != 1 else data_dict['names']  # class names
     assert len(names) == nc, f'{len(names)} names found for nc={nc} dataset in {data}'  # check
-    is_coco = data.endswith('coco.yaml') and nc == 80  # COCO dataset
-
-    # upload dataset to s3
-    if opt.upload_dataset and opt.s3_upload_dir:
-        with open(data, errors='ignore') as f:
-            data_dict = yaml.safe_load(f)  # load data dict
-        # upload yolo formatted data to s3
```

Comment:
this block is also important

Reply:
same comment here: if this code is not specific to the pip packaging, maybe it should move upstream?